src/static_analysis/manifest.rs

```diff
@@ -65,10 +65,10 @@ pub fn manifest_analysis<S: AsRef<str>>(config: &Config,
     results.set_app_label(manifest.get_label());
     results.set_app_description(manifest.get_description());
     results.set_app_version(manifest.get_version_str());
-    results.set_app_version_num(manifest.get_version_number());
-    results.set_app_min_sdk(manifest.get_min_sdk());
+    results.set_app_version_num(manifest.get_version_number() as i32);
+    results.set_app_min_sdk(manifest.get_min_sdk() as i32);
     if manifest.get_target_sdk().is_some() {
-        results.set_app_target_sdk(manifest.get_target_sdk().unwrap());
+        results.set_app_target_sdk(manifest.get_target_sdk().unwrap() as i32);
```

Comment:
Well, previously they were i32 and now are u32. I wanted to put a boundary on the changes I was doing. Anyway maybe is better to change the type on Results structure and the ones that uses Results. I'll change it